### PR TITLE
Update maintainers

### PR DESCRIPTION
**Pull request by mic-max**
## Changes

- Approver to Maintainer: @julianocosta89 , @mic-max 
- Maintainer to Approver: @austinlparker 
- Maintainer to Emeritus: @mtwo

A maintainer will need to make the accompanying changes to these teams

- https://github.com/orgs/open-telemetry/teams/demo-maintainers
- https://github.com/orgs/open-telemetry/teams/demo-approvers

PS: Is there a way we can just show all the members of a GitHub team nicely? Something similar to the contributors image. I guess it lacks the ability to see names when viewing the markdown though. Just thinking out loud 😂

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -164,21 +164,25 @@ For edit access, get in touch on
...
+- [Austin Parker](https://github.com/austinlparker), Lightstep
 - [Mikko Viitanen](https://github.com/mviitane), Dynatrace
 - [Penghan Wang](https://github.com/wph95), AppDynamics
 - [Reiley Yang](https://github.com/reyang), Microsoft
 - [Ziqi Zhao](https://github.com/fatsheep9146), Alibaba
 
+[Emeritus
+Maintainer/Approver/Triager](https://github.com/open-telemetry/community/blob/main/community-membership.md#emeritus-maintainerapprovertriager):
+
+- [Morgan McLean](https://github.com/mtwo), Splunk
```
Comment (reyang) [inline README.md:184]: @mtwo FYI.

**Comment by reyang** [comment]
Has this been discussed during the SIG meeting and does it follow https://github.com/open-telemetry/community/blob/main/community-membership.md#becoming-a-maintainer?

**Review by mtwo — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (mtwo) [reply README.md:184]: All good, I have a standing conflict with the weekly sync, so I haven't attended in a long time!

**Comment by austinlparker** [comment]
> Has this been discussed during the SIG meeting and does it follow https://github.com/open-telemetry/community/blob/main/community-membership.md#becoming-a-maintainer?

It hasn't been brought up during a SIG meeting, no. While this is technically within the guidance specified in the membership doc, it would have probably been good to discuss this with the existing maintainers before opening the PR.

**Review by austinlparker — CHANGES_REQUESTED** [review]
I suggest we discuss this at the next SIG meeting following KubeCon.

**Review by reyang — CHANGES_REQUESTED** [review]
https://github.com/open-telemetry/opentelemetry-demo/pull/411#issuecomment-1272011257 I would request at minimum a discussion among the maintainers before opening such kind of PR.

**Comment by cartersocha** [comment]
I agree we should cover this in the sig first. Let’s close for now.

**Pull request closed without merging**